Application/Relay/can_relay.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * can_relay.cpp
+ *
+ *  Created on: May 16, 2024
+ *      Author: Nico
+ */
+#include "./can_relay.hpp"
+namespace application{
+
+	Can_Relay::Can_Relay(std::shared_ptr<platform::ICan> can_bus,
+						 CircularQueue<DataPayload> queue):
+		can_bus_(can_bus),
+		queue_(queue)
+		{ messageSize = queue_.GetSize(); }
+
+	void Can_Relay::bitSet(float value, uint8_t* byteArray) {
+	    std::memcpy(byteArray, &value, sizeof(float));
+	    std::reverse(byteArray, byteArray + 4);
+	}
+
+	void Can_Relay::Generate_Messages(application::DataPayload data){
+		transmission_ended_ = false;
+		float row[messageSize] = {0};
+		uint8_t r;
+		uint8_t c;
```

Comment:
Consider more explicit variable names. I am not sure what these are referring to

Reply:
row and column, never updated from the rapid development when starting this feature, fixed